File: 백준/Bronze/1978. 소수 찾기/소수 찾기.cc
#include <iostream>
using namespace std;

int main() {
  int n, i, j;
  int s[101];
  int cnt = 0;

  cin >> n;

  for (i = 0; i < n; i++) {
    int tmp = 0;
    cin >> s[i];
    
    if (s[i] == 1)
      continue;

    for (j = 2; j < s[i]; j++) {
      if (s[i] % j == 0) {
        tmp += 1;
        break;
      }
    }
    if (tmp == 0)
      cnt += 1;
  }

  cout << cnt;
}

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.py
n = int(input())
s = list(map(int, input().split()))
cnt = 0

for i in range(n):
  tmp = 0
  if s[i] == 1:
    continue

  for j in range(2, s[i]):
    if s[i] % j == 0:
      tmp += 1
  if tmp == 0:
    cnt += 1

print(cnt)
